bokehjs/src/coffee/models/annotations/tooltip.coffee

```diff
@@ -61,6 +64,8 @@ export class TooltipView extends AnnotationView
 
     show(@el) # XXX: {offset,client}Width() gives 0 when display="none"
 
+    # slightly confusing: side "left" is relative to point that is being annotated
+    # but CS class "bk-left" is relative to the tooltip itself
```

Comment:
That's because `bk-left` relates to the arrow, so it's reversed in meaning to that of `attachment`'s `left`. Anyway, it's confusing.